controllers/login.js

```diff
@@ -234,11 +385,12 @@ router.get('/login/success', authHelper.authChecker, async (req, res, next) => {
 				},
 			});
 
-		if (consentStatus === 'ok' && haveBeenUpdated === false) {
+		if (consentStatus === 'ok' && haveBeenUpdated === false && (res.locals.currentUser.preferences || {}).firstLogin) {
```

Comment:
user is defined above as res.locals.currentUser, so you can shorten it to user.preferences